airgun/views/settings.py

```diff
@@ -9,6 +9,8 @@
 
 class SettingsView(BaseLoggedInView, SearchableViewMixin):
     title = Text("//h1[text()='Settings']")
+    email_button = Text('#test_mail_button')
```

Comment:
Use the Button widget, try to use XPath instead of CSS `#test_mail_button` for uniform purpose.

Reply:
You are right, the Button widget is the right way to do this. I will prefer id instead of XPath here.